README.md

```diff
@@ -26,9 +26,13 @@ A publisher can be any python program that imports a Kale-based task class and i
 Then the publisher publishes a task to Amazon SQS, which normally takes 10s miliseconds to return:
 
     import tasks
-    tasks.MyTask.publish(arg1, arg2)
+    tasks.MyTask.publish(None, arg1, arg2)
    
-The publish() function is a static method of a task class and it has the same signiture as the run\_task() method. A worker process, which may run on a different machine, will pick up the message and execute run\_task() method of the task.
+The publish() function is a static method of a task class.  Other than the first parameter, which can usually be None, it has the same signiture as the run\_task() method. A worker process, which may run on a different machine, will pick up the message and execute run\_task() method of the task.
```

Comment:
backticks will make make publish() a fixed-width font.  I like to use them for all code references.  `publish()` `run_task()`, etc.